code/clue.js

```diff
@@ -1,59 +1,350 @@
 // STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
 
 const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  firstName: "Jacob",
+  lastName: "Green",
+  color: "green",
+  description: "He has a lot of connections",
   age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  image: "assets/green.png",
+  occupation: "Entrepreneur",
+  favouriteWeapon: "knife",
+};
+
+const professorPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "He has a lot of medicin knowledge",
+  age: 40,
+  image: "assets/plum.png",
+  occupation: "professor",
+  favouriteWeapon: "rope",
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She is creative",
+  age: 35,
+  image: "assets/scarlet.png",
+  occupation: "architect",
+  favouriteWeapon: "candlestick",
+};
+
+const mrsPeackok = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She is free-minded",
+  age: 55,
+  image: "assets/peacock.png",
+  occupation: "actress",
+  favouriteWeapon: "dumbbell",
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He is strong",
+  age: 34,
+  image: "assets/mustard.png",
+  occupation: "butcher",
+  favouriteWeapon: "poison",
+};
+
+const mrsWhite = {
+  firstName: "Ana",
+  lastName: "White",
+  color: "white",
+  description: "She is very weak",
+  age: 28,
+  image: "assets/white.png",
+  occupation: "painter",
+  favouriteWeapon: "axe",
+};
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: "rope",
+  weight: 10,
+  id: "rope",
+};
+
+const knife = {
+  name: "knife",
+  weight: 5,
+  id: "knife",
+};
+
+const candlestick = {
+  name: "candlestick",
+  weight: 7,
+  id: "candlestick",
+};
+
+const dumbbell = {
+  name: "dumbbell",
+  weight: 30,
+  id: "dumbbell",
+};
+
+const poison = {
+  name: "poison",
+  weight: 2,
+  id: "poison",
+};
+
+const axe = {
+  name: "axe",
+  weight: 12,
+  id: "axe",
+};
+
+const bat = {
+  name: "bat",
+  weight: 13,
+  id: "bat",
+};
+
+const trophy = {
+  name: "trophy",
+  weight: 20,
+  id: "trophy",
+};
+
+const pistol = {
+  name: "pistol",
+  weight: 90,
+  id: "pistol",
+};
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
+console.log(colonelMustard.color);
+console.log(pistol.weight);
```

Comment:
These console logs are there for you, to check if your code works, right? In that case, you should delete them before handing in the project, I think.